topi/python/topi/math.py

```diff
@@ -612,6 +612,31 @@ def _compute(*indices):
         return tvm.te.max(tvm.te.min(value, const_max), const_min)
     return te.compute(x.shape, _compute)
 
+@tvm.te.tag_scope(tag=tag.ELEMWISE)
+def fixed_point_multiply(x, multiplier, shift):
+    """
+
+    Parameters
+    ----------
+    x :          tvm.te.Tensor or Expr
+                 Input argument.
```

Comment:
Indentation - There should be two tabs here. Seems many tabs.

Reply:
I reformatted it to follow the same style of other help texts.